definitions/npm/jest_v14.0.x/flow_>=v0.23.x/jest_v14.0.x.js

```diff
@@ -0,0 +1,71 @@
+// flow-typed signature: e2130120dcdc34bf09ff82449b0d508c
+// flow-typed version: 230d7577ce/jest_v12.0.x/flow_>=v0.23.x
+
+type JestMockFn = {
+  (...args: Array<any>): any;
+  mock: {
+    calls: Array<Array<any>>;
+    instances: any;
```

Comment:
look, I just copied what you already had there. Can you please just merge this and then follow-up with a fix?